README.md

```diff
@@ -25,7 +25,42 @@ xmlseclibs is being used in many different software.
 * [SimpleSAMLPHP](https://github.com/simplesamlphp/simplesamlphp)
 * [LightSAML](https://github.com/aerialship/lightsaml)
 
-
+## Basic usage
+
+The example below shows basic usage of xmlseclibs, with a SHA-256 signature.
+
+```php
+// Load the XML to be signed
+$doc = new DOMDocument();
+$doc->load('./path/to/file/tobesigned.xml');
+
+// Create a new Security object 
+$objDSig = new XMLSecurityDSig();
+// Use the c14n exclusive canonicalization
+$objDSig->setCanonicalMethod(XMLSecurityDSig::EXC_C14N);
+// Sign using SHA-256
+$objDSig->addReference($doc, XMLSecurityDSig::SHA256, array('http://www.w3.org/2000/09/xmldsig#enveloped-signature'));
```

Comment:
Split argument list in different lines for to avoid horizontal scrollbar when render